index.html

```diff
@@ -58,7 +58,10 @@
 
     // extend the bibliography entries
     localBiblio: ccg.localBiblio,
-    xref: "web-platform",
+    xref: ["web-platform", "vc-data-integrity", "controller-document"],
```

Comment:
```suggestion
    xref: ["web-platform", "vc-data-integrity", "controlled-identifiers"],
```

Not sure exactly what xref is doing here, but perhaps this should be updated to the new name?